package.json

```diff
@@ -96,9 +96,11 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.20.0",
+    "chalk": "^1.1.1",
```

Comment:
Jest uses this version, as does jsome - figured I might start colorizing text for local setup or for [danger init](https://github.com/danger/danger/blob/master/lib/danger/commands/init.rb#L28)